[2025-04-09T15:20:25.359+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T15:20:25.376+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-09T15:20:21.245907+00:00 [queued]>
[2025-04-09T15:20:25.385+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-09T15:20:21.245907+00:00 [queued]>
[2025-04-09T15:20:25.386+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-09T15:20:25.548+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_raw_data> on 2025-04-09 15:20:21.245907+00:00
[2025-04-09T15:20:25.556+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=93) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-09T15:20:25.558+0000] {standard_task_runner.py:72} INFO - Started process 97 to run task
[2025-04-09T15:20:25.559+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'extract_raw_data', 'manual__2025-04-09T15:20:21.245907+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmpia5wbp5l']
[2025-04-09T15:20:25.560+0000] {standard_task_runner.py:105} INFO - Job 61: Subtask extract_raw_data
[2025-04-09T15:20:25.664+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-09T15:20:21.245907+00:00 [running]> on host 5a8aeda502d0
[2025-04-09T15:20:25.863+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T15:20:21.245907+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T15:20:21.245907+00:00'
[2025-04-09T15:20:25.865+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-09T15:20:25.997+0000] {driven_data_pipeline.py:147} INFO - Started batch processing for 2025-04-09.
[2025-04-09T15:20:25.997+0000] {driven_data_pipeline.py:34} INFO - Created synthetic data for RO country code.
[2025-04-09T15:20:26.962+0000] {driven_data_pipeline.py:104} INFO - Written 586 records to the CSV file.
[2025-04-09T15:20:27.874+0000] {driven_data_pipeline.py:120} INFO - Added UUID to the dataset.
[2025-04-09T15:20:27.875+0000] {driven_data_pipeline.py:155} INFO - Finished batch processing 2025-04-09.
[2025-04-09T15:20:27.876+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-09T15:20:27.887+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-09T15:20:27.888+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=extract_raw_data, run_id=manual__2025-04-09T15:20:21.245907+00:00, execution_date=20250409T152021, start_date=20250409T152025, end_date=20250409T152027
[2025-04-09T15:20:28.027+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T15:20:28.057+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-09T15:20:28.092+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
